policyengine_uk/parameters/gov/dfe/targeted_childcare_entitlement/hours_entitlement.yaml

```
@@ -0,0 +1,10 @@
+description: The Department for Education considers this number of hours of free childcare provided by the targeted childcare entitlement.
```

```suggestion
description: The Department for Education provides this number of hours of free childcare under the targeted childcare entitlement.
```

Not sure the previous one made sense?

---

Edited.